src/Endpoints/Users/ReferenceNumber.php

```diff
@@ -21,14 +21,14 @@ class ReferenceNumber extends Entity
     use ReferenceNumberTrait;
 
     /**
-     * @Groups({"create", "get", "list", "update"})
+     * @Groups({"create"})
```

Comment:
why don't we move these properties to trait?

Reply:
good catch, fixing